docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -972,6 +972,11 @@ connector using a {doc}`WITH </sql/create-table-as>` clause.
   - Optionally specifies the format of table data files; either `PARQUET`,
     `ORC`, or `AVRO`. Defaults to the value of the `iceberg.file-format` catalog
     configuration property, which defaults to `PARQUET`.
+* - `compression-codec`
```

Comment:
I think table properties are underscored instead of hyphenated, the code seems to use `"compression_codec"` in `IcebergTableProperties`